Anchor.toml

```diff
@@ -8,8 +8,8 @@ resolution = true
 skip-lint = false
 
 [programs.localnet]
-fusion_swap = "HNarfxC3kYMMhFkxUFeYb8wHVdPzY5t9pupqW5fL2meM"
-whitelist = "5jzZhrzqkbdwp5d3J1XbmaXMRnqeXimM1mDMoGHyvR7S"
+fusion_swap = "12nFacVKzTjC5d82ZVV3Le9PFrnFw2vw2LR751MkXZFy"
```

Comment:
Is it really necessary to change the program ids?

Reply:
It seems like this is some local issue with your environment, because we tested it on two different machines, and anchor build works fine for us and doesn’t change the program IDs.

Our colleagues from other development teams sometimes check declare_id, so there should be a solid reason for it not to match the IDs used in the actually deployed programs. Could you please revert the IDs to the ones currently in the master branch?

Reply:
@SevenSwen Have reverted the program id changes in https://github.com/1inch/solana-fusion-protocol/pull/92/commits/810737e47e011544323776ada7c51f22449da02c